core/services/chainlink/testdata/config-full.toml

```diff
@@ -344,6 +344,22 @@ LeaseDuration = '0s'
 NodeIsSyncingEnabled = true
 FinalizedBlockPollInterval = '1s'
 
+[EVM.NodePool.Errors]
+NonceTooLow = ''
+NonceTooHigh = ''
+ReplacementTransactionUnderpriced = ''
+LimitReached = ''
+TransactionAlreadyInMempool = ''
+TerminallyUnderpriced = ''
+InsufficientEth = ''
+TxFeeExceedsCap = ''
+L2FeeTooLow = ''
+L2FeeTooHigh = ''
+L2Full = ''
+TransactionAlreadyMined = ''
+Fatal = ''
+ServiceUnavailable = ''
+
```

Comment:
This one is an input file, right? Should be able to omit these in that case
```suggestion
```

Reply:
Yeah we can remove this, I'll also have to remove the check for `AssertFieldsNotNil` since we will expect nil values for these strings.